website/src/components/Homepage/Header/HomepageHeader.module.scss

```diff
@@ -1,67 +1,76 @@
+@import '@site/src/css/fonts';
+
 .heroBanner {
+  --ifm-heading-color: #fff;
+  --ifm-button-background-color: #fff;
+  --ifm-color-gray-900: #12161B;
+
   height: 725px;
   width: 100%;
-  padding: 4rem 0;
-  color: white;
+  padding: 54px 10px;
+  color: var(--ifm-heading-color);
```

Comment:
Doesn't it have `--ifm-heading-color` by default already? Isn't that a tautology?

Reply:
Indeed, it can be removed freely.